### ISSUE TITLE
Question regarding the indicator lines

### ISSUE THREAD

**Issue by LouJay** [MGK82/PackTracker#9]
What do the orange, yellow and dashed blue lines mean? I understand that 39 is the maximum of packs without a legendary and in that case you should get one in the 40th, but what do the other lines track?

**Comment by MGK82** [comment]
Hi there. The yellow lines are static and indicate the average that Blizzard seems to aim on. It's the 5th pack for epic and the 20th for legendaries. So when you have 5 packs "without" an epic or 20 packs "without" a legendary, you missed that. That's why they are located on 4/19

The blue dashed line is your personal average. Since it doesn't make any sense to take your first 10 packs legendary into that average it ignores your first one. So don't worry when it's not shown for all sets yet. It also doesn't rise along with the bars. That means, it updates everytime you hit an epic or legendary, but not during your streaks towards one. If you want a "live" average where also your first legendary is included, please go to the statistic-windows. The statistic is not meant to interpred your pity counters and gives raw data back.

That should also be explained in the popups when you click on the question marks at the numeric counters. I feel very sorry that this is not clear enough for the people. I'll see if I can find a better way to signal this.

So long and if I can help you with something more, just let me know.

**Comment by LouJay** [comment]
Thanks for the response! I had clicked on the question marks you mention, but the popups seemed to be empty and I just thought they were placeholders for future stuff. I now realize that they weren't empty, but I just couldn't read them. I think I figured out what was happening: I'm using the Dark theme of HDT (setting in Appearance), and since it changes the background to black, the font is changed to white. However, the popups have a light background, and that remains the same, so the font in white becomes difficult to notice... I changed my theme to Light and was able to see the explanations. Thanks again! (I was starting to figure them out after a while, hehehe)
